recipes-tag/openmpi/meta.yaml

```diff
@@ -1,10 +1,10 @@
-# copied and pasted by Leo Fang from the following commit:
+# adapted by Leo Fang from the following commit:
 # https://github.com/conda-forge/openmpi-feedstock/blob/1d6390794529ad80f5b3416fa2cb98882d1c95fa/recipe/meta.yaml
 
-{% set version = "3.1.3" %}
+{% set version = "3.1.1" %}
 {% set sha256 = "8be04307c00f51401d3fb9d837321781ea7c79f2a5a4a2e5d4eaedc874087ab6" %}
 {% set major = version.rpartition('.')[0] %}
-{% set build = 1001 %}
+{% set build = 1000 %}
```

Comment:
Set the build number to `1000` here as we don't have any build for openmpi (I manually removed them).